components/Book/index.js

```diff
@@ -2,15 +2,15 @@ import Image from "next/image";
 
 export default function Book({ book }) {
   return (
```

Comment:
Just a little suggestion: As you access your book props 8 times in this code (e.g. book.title) you could destructure the book prop before your return statement:
const {imageURL, title, EAN, language, pagenumber, publication, price, description} = book

This way you don't always need to repeat "book." on each line and it looks a little cleaner
e.g <h3>{EAN}</h3>

Reply:
A very valuable suggestion, thank you 😊